src/main.rs

```diff
@@ -163,18 +163,22 @@ Faster Nmap scanning with Rust."#;
     println!("{}", info.gradient(Color::Yellow).bold());
     funny_opening!();
 
-    let config_path = match dirs::config_dir() {
-        Some(mut path) => {
-            path.push("rustscan");
-            path.push("config.toml");
-            path
-        }
-        None => panic!("Couldn't find config dir."),
+    let mut config_dir = match dirs::config_dir() {
```

Comment:
we only use home dir now, config + home are too complex. I probably forgot to remove this from my code, but Kaeros and I agreed :laughing:

Reply:
All right, removed from the places where it's checked.